arcgis-ios-sdk-samples/Maps/Mobile map (search and route)/MobileMapViewController.swift

```diff
@@ -60,15 +60,15 @@ class MobileMapViewController: UIViewController, AGSGeoViewTouchDelegate {
         // Dispose of any resources that can be recreated.
     }
     
-    private func symbolForStopGraphic(withIndex: Bool, index: Int?) -> AGSSymbol {
+    fileprivate func symbolForStopGraphic(_ withIndex: Bool, index: Int?) -> AGSSymbol {
```

Comment:
similar to other comments above about having argument name to clarify call site usage